requirements.txt

```diff
@@ -1,4 +1,4 @@
-gevent>=1.2.2
+gevent~=1.5.0
```

Comment:
please create separate PR or issue for that. This PR should really focus on bugfix and not update dependencies. It is not good idea to move too many things in one block.

Reply:
~= means that it is pinned to the latest version of the package, greater or equal to 1.5.0, but still in 1.5.*. I cannot test the code without this modification because it is giving me errors. So, I think this change should be included too.

Reply:
created #48 for this.

Reply:
> I cannot test the code without this modification because it is giving me errors. So, I think this change should be included too.

Not at all. So far in new clean isolated environment this does not produce any error with usage relevant to gevenet since it istall latest 20.5.0. I am seeing that for more than half a year on internal servers. The error may really be just your local environment. 

Fix that is essentially "different string replace" should imho not depend on upgrade of "coroutine networking library" - it is a problem of what is produced, not where it is used.

If it would be really problem I would still rather discuss and merge #48 first and have bugfix PR isolated of dependency change PR.

Reply:
created PR #52 for this change, after all comments in #48. Please comment there before merging. #52 should be merged before this one.